### PR TITLE
autoformat server files

### PR DESCRIPTION
**Pull request by samouri**
No functional changes.
PR was created by:
1. `./node_modules/.bin/prettier --write server/**/*.js`
2. using find/replace to separate out format from dependency docblocks
3. few manual fixes

each of those steps is a separate commit

### CODE REVIEW

File: server/sanitize/index.js [hunk 1]
```diff
@@ -1,19 +1,23 @@
 /**
  * Encode single characters with backslashes.
+ * 
+ *
```
Comment (samouri) [inline server/sanitize/index.js]: weird.

Comment (samouri) [reply server/sanitize/index.js]: I'll fix this file. the rest seem fine.

File: server/sanitize/index.js [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
  * Encode single characters with backslashes.
+ *
  * @param {string} charStr single-character string.
  * @return {string} backslash escaped character.
- *
  * @copyright (c) 2013, GoInstant Inc., a salesforce.com company.
  * @license See CREDITS.md.
- * @see https://github.com/goinstant/secure-filters/blob/master/lib/secure-filters.js
- *
+ * @see https:
```
Comment (gziolo) [inline server/sanitize/index.js]: Url was cut here, bery strange bug.

File: server/state-cache/index.js [hunk 3]
```diff
@@ -1,12 +1,15 @@
 /**
  * External Dependencies
+ *
+ * @format
  */
+
```
Comment (gziolo) [inline server/state-cache/index.js]: I think those blank lines are obsolete after the comment block.

File: server/sanitize/index.js (hunk 2 above)
Comment (samouri) [reply server/sanitize/index.js]: nice catch, that bug may actually exist upstream in jest-docblock.  I'll investigate

File: server/state-cache/index.js (hunk 3 above)
Comment (samouri) [reply server/state-cache/index.js]: right now the desired format was actually to add in that newlines 🤷‍♂️.  I'm not passionate about this

Comment (gziolo) [reply server/state-cache/index.js]: I see... okey

File: server/sanitize/index.js (hunk 2 above)
Comment (samouri) [reply server/sanitize/index.js]: I've confirmed its a bug that it is unintended behavior of  `jest-docblock`.  I've opened a pr there: https://github.com/facebook/jest/pull/4837

**Comment by samouri** [comment]
no unit tests errors. eslint errors are:
```

/home/ubuntu/wp-calypso/server/bundler/loader.js
  142:2  error  Unexpected var, use let or const instead  no-var

/home/ubuntu/wp-calypso/server/bundler/css-hot-reload.js
  102:3  error  Unexpected var, use let or const instead  no-var

/home/ubuntu/wp-calypso/server/config/parser.js
  28:2  error  Unexpected var, use let or const instead  no-var


/home/ubuntu/wp-calypso/server/isomorphic-routing/index.js
  89:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs

/home/ubuntu/wp-calypso/server/config/validate-config-keys.js
  60:6  error  Strings must use singlequote  quotes


/home/ubuntu/wp-calypso/server/api/oauth.js
  29:2  error  Unexpected var, use let or const instead  no-var
  92:2  error  Unexpected var, use let or const instead  no-var

/home/ubuntu/wp-calypso/server/isomorphic-routing/loader.js
  1:1  error  Missing JSDoc @returns for function       valid-jsdoc
  1:1  error  Missing JSDoc for parameter 'sections'    valid-jsdoc
  3:2  error  Unexpected var, use let or const instead  no-var
```

**Comment by samouri** [comment]
now that the jest pr got merged, and we came up with a good method for separating out pragamas from docblocks,  I'm happy with this PR.

@gziolo: you think this looks mergeable?

File: server/isomorphic-routing/loader.js [hunk 4]
```diff
@@ -1,8 +1,14 @@
+/** @format */
 function getSectionsModule( sections ) {
-	var caseSections = ''
+	var caseSections = '';
 	sections.forEach( function( section ) {
 		if ( section.isomorphic ) {
-			caseSections += 'case ' + JSON.stringify( section.module ) + ': return require( ' + JSON.stringify( section.module ) + ' );\n';
+			caseSections +=
```
Comment (samouri) [inline server/isomorphic-routing/loader.js]: i'm a big fan of this one

File: server/sanitize/index.js [hunk 5]
```diff
@@ -1,19 +1,22 @@
...
- **/
+ */
+
 function jsSlashEncoder( charStr ) {
 	var code = charStr.charCodeAt( 0 ),
 		hex = code.toString( 16 ).toUpperCase();
 
-	if ( code < 0x80 ) { // ASCII
+	if ( code < 0x80 ) {
+		// ASCII
```
Comment (gziolo) [inline server/sanitize/index.js]: I guess this comment should go before if statement now. It isn't prettier's fault though :)

**Comment by gziolo** [comment]
Looks legit! 👍 

It would be a good idea to test if docker still builds production version properly :)

**Comment by samouri** [comment]
rebased

File: server/bundler/loader.js [hunk 6]
```diff
@@ -35,11 +36,11 @@ function getSectionsModule( sections ) {
 			'	var loadCSS = function( id, urls ) {',
 			'		var url = urls.ltr;',
 			'',
-			'		if ( typeof document !== \'undefined\' && document.documentElement.dir === \'rtl\' ) {',
+			"		if ( typeof document !== 'undefined' && document.documentElement.dir === 'rtl' ) {",
```
Comment (samouri) [inline server/bundler/loader.js]: not a bad change to remove the escapes

**Pull request merged**